tests/unit/components/UploadHistory.spec.js

```diff
@@ -17,15 +17,15 @@ describe('UploadHistory.vue', () => {
     const wrapper = mount(UploadHistory, {
       store,
       localVue,
-      stubs: ['router-link'],
-      propsData: {
-        uploads: [{ filename: 'one.xlsx' }, { filename: 'two.xlsx' }]
-      }
+      stubs: ['router-link']
     })
-    expect(wrapper.text()).to.include('Upload')
+    expect(wrapper.text()).to.include('No  uploads')
+
+    /* TODO: figure out how to intercept fetch api requests
```

Comment:
If we're consistent, I think we should be able to mock this file in tests:

https://github.com/usdigitalresponse/arpa-reporter/blob/staging/src/store/index.js

I'm not as familiar with mocha+chai, but it looks like this plugin might be the way to go?

https://www.chaijs.com/plugins/chai-spies/